### DISCUSSION TITLE
GW 4.1 - 3,5" drive LED still on when connected

### DISCUSSION THREAD

**Discussion by mathe1** [keirf/greaseweazle#401] [Q&A] [answered]
Hi, 

I first time buyed the GW 4.1
Delivered with jumper set on "USB 5V enabled" and an "Write enable".

I connect the (twisted) floppy cable, plug the power cable (first from GW, then with power adapter (maybe it need 12V too) - same problem).

Normally it should short time run then stop and LED off, I think.
But with my GW the drive is still running, LED is on all the time. It is not so fine for diskette when inserted while the drive is running...

I tried an other 3,5" floppy drive, too, and an other floppy cable - but same issue.

What's problem?

**Comment by keirf** [comment]
Probably the ribbon cable is upside down and needs flipping at one of the two ends.

**Reply by mathe1** [reply]
No, I can't plug it in the other way around because it has this cam in the middle; and the red marked wire is where the triangle is on the board. In same way on the drive.

**Reply by keirf** [reply]
So can only be inserted "right way round" at each end. Usually pin 1 is to left hand side as you look at the back of floppy drive, and left hand side as you look top down at the Greaseweazle.

Does the Greaseweazle work except for motor always on?

**Reply by keirf** [reply]
Are the drives PC drives (drive select 1)? Are the ribbons straight or with a twist?

**Reply by keirf** [reply]
Is the motor always on, or does it simply linger after an operation?

**Reply by mathe1** [reply]
The motor is always on and the LED too.
Even I disconnect GW from USB-C (when not use the USB power).

Only when disconnect floppy wire, the device motor stops, LED off (when the device is powered by external PSU, not like in the photo).

![31608e6f-8e05-4b26-b641-410ed69c82a9](https://github.com/keirf/greaseweazle/assets/7896874/9115a81c-97fb-49ed-8d94-4019c75f9459)

This drive seems be a Sony 8712 (or 18Z7G - printed on motor)
Other drive I have is "Mitsumi model D359M3".

**Reply by keirf** [reply, marked as answer]
Unusual for a drive to have the connector that way round. Is your other drive the same? 

If the connector is wrong way round you get exactly this symptom and of course the drive will not work (because all signal lines are grounded).

All odd pins are ground and all even are signals so you can usually also confirm things with a look at what pins are tied together at the drive connector, for example.

**Reply by mathe1** [reply]
Yes,

I checked again on a 3rd drive, there is a space also on lower side to plug the connector in the other way.

Now it works - have a lot of thanks for your support.

**Reply by ourIThome** [reply]
I have a Panasonic drive where the key is on the wrong side... So a sharp knife and cut a new key in the plastic on the drive.  Here's a photo... Don't always trust the key...

![PXL_20240125_204437461.jpg](https://github.com/keirf/greaseweazle/assets/13407658/fdf8c67f-4a95-4c00-a62c-729ff1a6889a)
